src/screens/SignUp/index.tsx

```diff
@@ -181,7 +181,7 @@ const SignUp = () => {
 
   return (
     <ScreenWrapper>
-      <VStack>
+      <VStack px={8}>
```

Comment:
avoid magical numbers. Use constants from src/styles/theme.ts instead

Reply:
poderia vir deixar uma santa maria premium aqui no 902 TO

Reply:
I created a file to store sizes constants